pkg/bgpv1/agent/controller.go

```diff
@@ -391,3 +395,20 @@ func (c *Controller) Reconcile(ctx context.Context) error {
 func (c *Controller) FullWithdrawal(ctx context.Context) {
 	_ = c.BGPMgr.ConfigurePeers(ctx, nil, nil) // cannot fail, no need for error handling
 }
+
+// applyPolicyDefaults applies default values on the CiliumBGPPeeringPolicy.
+func (c *Controller) applyPolicyDefaults(policy *v2alpha1api.CiliumBGPPeeringPolicy) {
+	for _, r := range policy.Spec.VirtualRouters {
+		for _, n := range r.Neighbors {
+			if n.ConnectRetryTime.Duration == 0 {
+				n.ConnectRetryTime.Duration = types.DefaultBGPConnectRetryTime
+			}
+			if n.HoldTime.Duration == 0 {
```

Comment:
RFC4271 Sec 4.4 says that hold time = 0 has a special meaning that disables keepalive. I'm not sure how practical it is to use that, but how about making the CRD field `nullable` and checking whether it is nil here instead of checking zero?